Code/UnitTests/RemoteTestHarness/CMakeLists.txt

```diff
@@ -1,5 +1,7 @@
 ez_cmake_init()
 
+ez_build_filter_everything()
+ez_requires_qt()
```

Comment:
Does this require QT? I thought this is a C# app. @Sanakan8472

Reply:
It does, without the ez_requires_qt there are cmake errors when certain other projects are filtered out. I actually talked to @Sanakan8472 back when I hit this issue.